exercises/concept/language-list/.meta/examples/languageList.example.u

```diff
@@ -0,0 +1,26 @@
+languageList.new : [Text]
+languageList.new = []
+
+languageList.add : Text -> [Text] -> [Text]
+languageList.add input list = input +: list
+
+languageList.drop1 : [Text] -> [Text]
+languageList.drop1 = cases
+  head +: tail -> tail
+  [] -> []
+
+languageList.size : [Text] -> Nat
+languageList.size list =
+  go acc = cases
+    [] -> acc
+    h :+ t -> go (Nat.increment acc) t
+  go 0 list
+
+languageList.contains : Text -> [Text] -> Boolean
+languageList.contains input = cases
+  [] -> false
+  h +: t | h === input -> true
+  h +: t -> contains input t
```

Comment:
This uses a guard pattern, which is likely either a separate prerequisite or part of a `pattern-matching` prerequisite.